Sources/Navidux/Core/Navigation/NavigationAction/Actions/PushNavigationAction.swift

```diff
@@ -11,6 +11,14 @@ public struct PushNavigationAction: NavigationAction {
     public enum PresentationStyle {
         case fullScreen
         case modal(completion: (() -> Void)?)
+        case bottomSheet(BottomSheetSize, completion: (() -> Void)?)
+    }
+    
+    public enum BottomSheetSize {
+        case fixed(CGFloat)
+        case halfScreen
+        case fullScreen
+        case auto
     }
```

Comment:
I think we have to move this from `PushNavigationAction` and make it declared on its own context or inside `BSPresentationController`

Reply:
Done